### DISCUSSION TITLE
1Gram+2Gswap被oomkill了

### DISCUSSION THREAD

**Discussion by Miuzarte** [FloatTech/ZeroBot-Plugin#584] [General]
感觉不太正常？平时ram占用都只有50%，刚刚突然发现bot没反应了上去看到killed了
版本是v1.6.2-beta3
![QQ截图20230213020424](https://user-images.githubusercontent.com/66856838/218329494-5b14445f-f23e-4222-8f0c-2fd7d2dcc5bb.png)
![image](https://user-images.githubusercontent.com/66856838/218329476-215ef450-1d77-414e-9552-6182e949e35f.png)

**Comment by Miuzarte** [comment]
又出现了一次  好像是查弹幕的锅？
![image](https://user-images.githubusercontent.com/66856838/218417818-36dccfd5-2dd5-476e-b5c7-6187f46156b7.png)
![image](https://user-images.githubusercontent.com/66856838/218418053-c8a6ce02-0781-40f7-bdcb-68d45c6b49e1.png)

**Comment by Jiang-Red** [comment]
查弹幕的制图会占用大量资源
